### ISSUE TITLE
Some license files differ in name only via CASE

### ISSUE THREAD

**Issue by dutchgecko** [ryanoasis/nerd-fonts#468]
#### 🎯 Subject of the issue

In #463 license files were added to all of the patched fonts. However, some files differ in name only via their case, for example `patched-fonts/Agave/complete/LICENSE.TXT` vs `patched-fonts/Agave/complete/LICENSE.txt`. This causes an incomplete clone on case-insensitive filesystems such as under Windows. Git also really struggles, and produces odd behaviour.

#### 🔧 Your Setup

- Windows 10

#### ★ Optional

```git
On branch master
Your branch is up to date with 'origin/master'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

        modified:   patched-fonts/Agave/complete/LICENSE.TXT
        modified:   patched-fonts/Agave/complete/LICENSE.txt
        modified:   patched-fonts/AnonymousPro/complete/LICENSE.TXT
        modified:   patched-fonts/AnonymousPro/complete/LICENSE.txt
        modified:   patched-fonts/AurulentSansMono/complete/LICENSE.TXT
        modified:   patched-fonts/AurulentSansMono/complete/LICENSE.txt
        modified:   patched-fonts/BigBlueTerminal/complete/LICENSE.TXT
        modified:   patched-fonts/BigBlueTerminal/complete/LICENSE.txt
        modified:   patched-fonts/CascadiaCode/complete/LICENSE.TXT
        modified:   patched-fonts/CascadiaCode/complete/LICENSE.txt
        modified:   patched-fonts/DaddyTimeMono/complete/LICENSE.TXT
        modified:   patched-fonts/DaddyTimeMono/complete/LICENSE.txt
        modified:   patched-fonts/DroidSansMono/complete/LICENSE.TXT
        modified:   patched-fonts/DroidSansMono/complete/LICENSE.txt
        modified:   patched-fonts/HeavyData/complete/LICENSE.TXT
        modified:   patched-fonts/HeavyData/complete/LICENSE.txt
        modified:   patched-fonts/Inconsolata/complete/LICENSE.TXT
        modified:   patched-fonts/Inconsolata/complete/LICENSE.txt
        modified:   patched-fonts/LiberationMono/complete/LICENSE.TXT
        modified:   patched-fonts/LiberationMono/complete/LICENSE.txt
        modified:   patched-fonts/ShareTechMono/complete/LICENSE.TXT
        modified:   patched-fonts/ShareTechMono/complete/LICENSE.txt
```

```diff
diff --git a/patched-fonts/Agave/complete/LICENSE.TXT b/patched-fonts/Agave/complete/LICENSE.TXT
index fd662a7..5f04ebc 100644
--- a/patched-fonts/Agave/complete/LICENSE.TXT
+++ b/patched-fonts/Agave/complete/LICENSE.TXT
@@ -1,428 +1,33 @@
-Attribution-ShareAlike 4.0 International
-
-=======================================================================
-
-Creative Commons Corporation ("Creative Commons") is not a law firm and
-does not provide legal services or legal advice. Distribution of
-Creative Commons public licenses does not create a lawyer-client or
-other relationship. Creative Commons makes its licenses and related
[...]
+© 2014 Sam Radian. All Rights Reserved. 
+***
+C is either a registered trademark or a trademark of C font family in the Sri Lanka and/or other countries.
+***
+Code New Roman is aimed for use in programming environments and other circumstances where a monospaced font is specified. All characters have the same width, like old typewriters, making it a good choice for personal and business correspondance. This font improved and based on Mac OS Western New Roman,MONACO, Windows Consolas/Lusida Consolas, ABeeZee and Bitstream Vera Sans Mono.This font display allowed a design with proportions closer to normal text than traditional monospaced fonts like Courier. This allows for more comfortably reading of extended text on screen. OpenType features include hanging or lining numerals; slashed, dotted and normal zeros; and alternative shapes for a number of lowercase letters. The look of text can be tuned to personal taste by varying the number of bars and waves.
```

^ git has managed to check the wrong file out into this location

**Comment by mrvkino** [comment]
I have the same issue in macOS git is not able to manage this change correctly. I cannot restore, add or reset the change.

**Comment by BPDunbar** [comment]
It's been nearly a year now and this still hasn't been fixed, They are not duplicates and should be distinguished by somerthing other than case, possibly appending the licence type to the name. e.g. License (Attribution-ShareAlike 4.0 International).txt. 

Example

 nerd-fonts/patched-fonts/Agave/complete/LICENSE.TXT
is an Attribution-ShareAlike 4.0 International licence

 nerd-fonts/patched-fonts/Agave/complete/LICENSE.txt 
is an  Apache License Version 2.0

for completeness

 nerd-fonts/patched-fonts/Agave/complete/LICENSE 

is an MIT licence.

**Comment by Finii** [comment]
Has been fixed in 2f429c8facf175aded856a32423e41abe6d94b6f
